about.html

```diff
@@ -0,0 +1,143 @@
+            <li class="big-menu-right">
+              <a href="about.html">About</a>
+            </li>
+            <li class="big-menu-right">
+              <a href="index.html">Program</a>
+            </li>
+            <li class="big-menu-right">
+              <a href="index.html">Join</a>
+            </li>
+            <li class="big-menu-right">
+              <a href="index.html">Sponsors</a>
+            </li>
+            <li class="big-menu-right">
+              <a href="index.html">News</a>
+            </li>
+            <li class="big-menu-right"><button class="contact">Contact Us</button>
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <!-- Top Menu Desktop-->
+    </header>
+  </div>
+  <section>
+    <br>
+    <h1 class="event-name">Energy Conference<br>
+      Goiânia, Brazil</h1>
+    <p class="event-description mobile-none">
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Natus, minima saepe incidunt quis ex eum voluptatem rem
+      unde fuga repudiandae iure? Animi porro laboriosam odio dolore quisquam nam veritatis iste.<br><br> Lorem ipsum
+      dolor sit amet consectetur adipisicing elit. Non deleniti laboriosam quas, consequuntur sint hic adipisci
+      perferendis nisi corrupti odit dolores similique fugit minus libero optio quos rerum suscipit quae.<br><br>
+      Similique corporis in illo delectus! Commodi ducimus cum quam expedita, qui sit nisi itaque, asperiores officiis
+      facilis in maxime nesciunt voluptatibus. Quaerat debitis ea quidem? Minus, cupiditate? Soluta, eaque
+      dolorum.<br><br>
+      Ea quos labore officiis explicabo possimus? Suscipit beatae aliquid ipsam ab nisi perferendis aut magni! A quos
+      voluptate, cum eius, dicta quisquam sapiente dolor ea possimus, dignissimos quod delectus ipsa.<br><br>
+      Fugiat facere, dolorum cumque provident nisi dolore. Quo tempora suscipit accusantium perspiciatis quos aliquid
+      tenetur inventore ullam ut quae? Vel optio inventore dignissimos repellat unde magni beatae! Molestiae, optio ab?
+    </p>
+    <p class="event-description desktop-none">Lorem ipsum dolor sit amet consectetur adipisicing elit. Natus, minima
+      saepe incidunt quis ex eum voluptatem rem unde fuga repudiandae iure? Animi porro laboriosam odio dolore quisquam
+      nam veritatis iste.<br><br> Lorem ipsum dolor sit amet consectetur adipisicing elit. Non deleniti laboriosam
+      quas, consequuntur sint hic adipisci perferendis nisi corrupti odit dolores similique fugit minus libero optio
+      quos rerum suscipit quae.<br><br></p>
+  </section>
+  <!-- Contact-->
+  <section>
+    <div class="contact-event">
+      <h3 class="contact-event_title">
+        For more information about the Energy event, please contact us:
+      </h3>
+      <a href="mailto:name@email.com" class="contact-event_mail">energy2024@events.com</a>
+    </div>
+  </section>
+  <section class="logo-info">
+    <h2>World Energy</h2>
```

Comment:
- [ ] You are missing a section that contains a bottom section with a centered column that includes a section header, a divider, a line of text, and two images with text centered over them where the images are arranged vertically on mobile view and horizontally on desktop view. Please fix that.

Reply:
Sections added